### PR TITLE
wol: brew audit --strict; change homepage to more appropriate one

### PR DESCRIPTION
**Pull request by L2G**

### CODE REVIEW

**Comment by L2G** [comment]
Where the heck did "fernflower" come from…?

**Comment by L2G** [comment]
Is this "fernflower" the same one from #119?

File: wol.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
-require 'formula'
-
 class Wol < Formula
-  head 'https://github.com/kylef/wol.git'
-  homepage 'http://kylefuller.co.uk/'
+  head "https://github.com/kylef/wol.git"
```
Comment (DomT4) [inline wol.rb]: Likewise, let's flip `homepage` and `head` around here, and adding a `desc` would be :birthday: as well.

**Comment by DomT4** [comment]
Ping :wink:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-head-only/commit/4fa01039aef7bc9fc37d5ef492d6bc4bfd079b54. Thanks @L2G! :beers:

**Pull request closed without merging**